src/display/api.js

```diff
@@ -965,6 +976,11 @@ class PDFDocumentProxy {
  *                    image). The default value is 'rgb(255,255,255)'.
  * @property {Object} [annotationStorage] - Storage for annotation data in
  *                    forms.
+ * @property {Promise} [optionalContentConfigPromise] - A promise that should
```

Comment:
Once #12156 lands, this should be changed to mention the promise type and to have each line start with two spaces as per the standard.